packages/appstore/src/hooks/use-tradershub-tracking.ts

```diff
@@ -0,0 +1,113 @@
+import { Analytics } from '@deriv/analytics';
+import { useStore } from '@deriv/stores';
+import { useCallback, useEffect, useMemo } from 'react';
+
+// This hook is used to track the onboarding form in TradersHub
+export const useTradersHubTracking = () => {
+    const { traders_hub, ui, client } = useStore();
+
+    const { is_mobile } = ui;
+
+    const { loginid } = client;
+
+    const { is_first_time_visit } = traders_hub;
```

Comment:
is it possible to pass the things to the hook instead of subscription to the store?
this way moving forward if we want to remove the stores that would be easier.

Reply:
so you mean by passing these values from the function body where we are using this hook ? 

like:

```js
const { trackOnboardingClose } = useTradershubTracking({loginid, is_first_time_visit});
```
@ali-hosseini-deriv

Reply:
yes exactly.